### PR TITLE
[DL-9385] - update dependencies

### PR DESCRIPTION
**Pull request by peckover**

### CODE REVIEW

File: app/views/components/language_selection.scala.html [hunk 1]
```diff
@@ -1,31 +0,0 @@
-@*
- * Copyright 2022 HM Revenue & Customs
- *
```
Comment (derickdamoah) [inline app/views/components/language_selection.scala.html:3]: why was this file deleted?

Comment (peckover) [reply app/views/components/language_selection.scala.html:3]: it wasn't being used and all it was basically a wrapper for the now deprecated uk.gov.hmrc.urls.Link component

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -38,8 +40,8 @@ private object AppDependencies {
       override lazy val test = Seq(
         "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0" % scope,
```
Comment (derickdamoah) [inline project/AppDependencies.scala:41]: This can be increase to version 6.0.0
https://mvnrepository.com/artifact/org.scalatestplus.play/scalatestplus-play

File: project/scaffold.sbt [hunk 3]
```diff
@@ -1 +1 @@
-addSbtPlugin("org.foundweekends.giter8" % "sbt-giter8-scaffold" % "0.13.0")
\ No newline at end of file
+addSbtPlugin("org.foundweekends.giter8" % "sbt-giter8-scaffold" % "0.13.1")
```
Comment (derickdamoah) [inline project/scaffold.sbt:1]: version 0.15.0 is available

File: project/AppDependencies.scala (hunk 2 above)
Comment (peckover) [reply project/AppDependencies.scala:41]: That's a pre-release version. We don't usually use them because they might not be stable and other dependencies are  likely to be not compatible yet

File: project/scaffold.sbt (hunk 3 above)
Comment (peckover) [reply project/scaffold.sbt:1]: I tried to upgrade this further but it was incompatible with other dependencies. Can you get that version to work locally?

File: project/AppDependencies.scala (hunk 2 above)
Comment (derickdamoah) [reply project/AppDependencies.scala:41]: ok

File: project/scaffold.sbt (hunk 3 above)
Comment (derickdamoah) [reply project/scaffold.sbt:1]: No it doesnt work on mine either

**Review by derickdamoah — APPROVED** [review]
